recipes/xonda/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "xonda" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "09eefdf61bc987d1b1522e49fe116650b93d1536c47881ea7a197398beabaffe" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/gforsyth/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install
```

Comment:
When using `setuptools` you can do `python setup.py install --single-version-externally-managed --record record.txt`. Or better yet, change `setuptools` to `pip` and use `python -m pip install --no-deps --ignore-installed .`

Reply:
Pip/setuptools tends to fail for xonsh xontribs because it can't handle implicit namespace packages

Reply:
Good to know. However, more most Python packages in `conda-forge` I do recommend moving to `pip`, after all it is the standard :wink:

Reply:
Yeah it is just unfortunate that the standard does not adhere to the standard library :wink: